api/app_analytics/split_testing/views.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+from django.http import Http404
+from rest_framework import viewsets
+
+from environments.authentication import EnvironmentKeyAuthentication
+from environments.permissions.permissions import EnvironmentKeyPermissions
+
+from .models import SplitTest
+from .permissions import SplitTestPermissions
+from .serializers import ConversionEventSerializer, SplitTestSerializer
+
+
+class ConversionEventViewSet(viewsets.ModelViewSet):
+    permission_classes = (EnvironmentKeyPermissions,)
+    authentication_classes = (EnvironmentKeyAuthentication,)
+    serializer_class = ConversionEventSerializer
+    http_method_names = ["post"]
+
+
+class SplitTestViewSet(viewsets.ModelViewSet):
+    permission_classes = (SplitTestPermissions,)
```

Comment:
This should probably still also include `IsAuthenticated`

Reply:
As discussed briefly during standup, `IsAuthenticated` is the parent class of the `SplitTestPermissions`